pkg/dwarf/frame/entries.go

```diff
@@ -142,7 +144,7 @@ func (ptrEnc ptrEnc) Supported() bool {
 			// These values aren't defined at the moment
 			return false
 		}
-		if ptrEnc&0xf0 != ptrEncPCRel {
+		if (ptrEnc&0xf0)&^ptrEncSupportedFlags != 0 {
```

Comment:
While we're touching this code, can we assign the integer immediate values to constants and explain their usage.

Reply:
Done.